Ds3/Calls/DeleteDataPersistenceRuleSpectraS3Request.cs

```diff
@@ -47,7 +54,7 @@ internal override string Path
         {
             get
             {
-                return "/_rest_/data_persistence_rule/" + DataPersistenceRule;
+                return "/_rest_/data_persistence_rule/" + DataPersistenceRuleId.ToString();
```

Comment:
No its not. It does not cause any errors. Its present in other files and is not directly related to the updating DataPersistenceRule to be a UUID.  If you want, I can debug in a separate PR.